circle.yml

```diff
@@ -73,7 +74,7 @@ jobs:
             docker-compose exec worker supervisorctl restart all
             docker-compose exec web make -f Makefile-docker update_deps
             docker-compose exec web supervisorctl restart all
-            docker-compose exec web bash /code/scripts/ui-test.sh
+            docker-compose exec selenium-firefox tox -e ui-tests
```

Comment:
How does this work? We never install any dependencies in the selenium-firefox container or am I missing something?

Reply:
Ah, I just noticed https://github.com/jrbenny35/selenium-firefox/blob/master/Dockerfile#L33 so that's how it's working. I'm not sure I like the idea of it and I'm not sure if there's any actual difference from running the tests on the selenium box versus the web container? Both should be able to communicate back and forth just fine?

I'm fine with keeping it this way though if it works and allows us to improve incrementally.

Reply:
Very fair point, mind documenting this there so that we know why we do that in a few months time too?

Reply:
Will do